.circleci/config.yml

```diff
@@ -84,6 +84,44 @@ jobs:
           failure_message: Nightly << parameters.stack >> auth tests failed!
           webhook: $SLACK_WEBHOOK
 
+  audit:
+    working_directory: ~/repo
+    docker:
+      - image: cimg/base:stable-20.04
+        auth:
+          username: dockstoretestuser
+          password: $DOCKERHUB_PASSWORD
+    steps:
+      - browser-tools/install-browser-tools
+      - checkout
+      - run:
+          name: Checkout merge commit (PRs only)
+          command: |
+            if [[ -n "${CIRCLE_PULL_REQUEST}" ]]
+            then
+              git fetch origin +refs/pull/${CIRCLE_PULL_REQUEST##*/}/merge:
+              git checkout -qf FETCH_HEAD
+            fi
+      - install_container_dependencies
+      - run:
+          name: Run npm audit
+          #Only run 'npm ci' if node_modules was not restored from cache
+          command: |
+            bash -i -c "npm i"
+            bash -i -c "npm audit > vulnerabilities.txt" || true
+            bash scripts/npm-audit.sh
+            git stash
+            git checkout develop
+            bash -i -c 'npm i'
+            bash -i -c 'npm audit > vulnerabilities.txt' || true
```

Comment:
Could you instead just commit the vulnerabilities.txt file to develop and compare with that?

On the one hand, this makes our vulnerabilities visible to all. On  the other hand, anybody could run `npm audit` themselves and figure it out very easily.

I'm going to say no to this idea, but I'll leave in the comment in case anybody else likes it. :)

Reply:
:thinking: you as in circle automatically checks in and commits, or developer managed like the third party licenses file?

Reply:
I was thinking like the third party licenses file.